_data/projects/js_arules.yml

```
@@ -0,0 +1,10 @@
+name: JS Arules
+desc: Local Association Learning in your Browser using JS
+site: https://bacloud14.github.io/js_arules/
+tags:
+  - arules
+  - javascript
+  - visualization
+upforgrabs:
+  name: up for grabs
```

This needs to exactly match the tag used on the project:

```suggestion
  name: up-for-grabs
```

---

I did not pay attention. Many thanks @shiftkey